render-utils.js

```diff
@@ -0,0 +1,42 @@
+export function renderPokemonCard(pokemon) {
+    const div = document.createElement('div');
+    const nameEl = document.createElement('div');
+    const IdNum = document.createElement('div');
```

Comment:
try to use camelCase for all variables ie. `const idNum`. I know it feels weird bc id is capital XD.